src/cmd/oui.rs

```diff
@@ -66,11 +70,92 @@ pub struct Submit {
     commit: bool,
 }
 
+/// Updates an organizational OUI. The transaction is not
+/// submitted to the system unless the '--commit' option is given.
+#[derive(Debug, StructOpt)]
+pub enum Update {
+    /// The address(es) of the router to send packets to. This will overwrite any previous
+    /// routers
+    Routers(update::Routers),
+    /// If less than 5 filters have been defined,
+    /// you can create an additional Xor
+    NewXor(update::NewXor),
+    UpdateXor(update::UpdateXor),
+    /// Requested additional subnet size. Must be a value between 8 and 65,536
+    /// and a power of two.
+    RequestSubset(update::RequestSubet),
+}
+
+mod update {
+    use super::{PublicKey, StructOpt};
+    #[derive(Debug, StructOpt)]
+    pub struct Routers {
+        /// OUI to update
+        #[structopt(long)]
+        pub oui: u32,
+        /// The address(es) of the router to send packets to
+        #[structopt(long = "address", short = "a", number_of_values(1))]
+        pub addresses: Vec<PublicKey>,
+        /// Which OUI nonce this transaction has
+        #[structopt(long)]
+        pub nonce: Option<u64>,
+        /// Commit the transaction to the API
+        #[structopt(long)]
+        pub commit: bool,
+    }
+    #[derive(Debug, StructOpt)]
+    /// Update an already defined Xor
+    pub struct UpdateXor {
```

Comment:
Probably just a nit but why not split up xor update/create in sub-commands? like `oui update xor create` and `oui update xor update`?

Reply:
Could do. I made this reflect the proto enums too much, I think.

Reply:
Fixed.